exec/java-exec/src/main/java/org/apache/drill/exec/work/foreman/QueryManager.java

```diff
@@ -81,6 +85,7 @@
   private final String stringQueryId;
   private final RunQuery runQuery;
   private final Foreman foreman;
+  private final List<Option> queryOptions;
```

Comment:
Not required. Could use `foreman.getQueryContext().getOptionManager().getOptionList()`?